src/validation.cpp

```diff
@@ -227,6 +227,7 @@ CAmount maxTxFee = DEFAULT_TRANSACTION_MAXFEE;
 
 CBlockPolicyEstimator feeEstimator;
 CTxMemPool mempool(&feeEstimator);
+std::atomic_bool g_is_mempool_loaded(false);
```

Comment:
style-nit: Might use `g_is_mempool_loaded{false}` for compile-time narrowing check.

Reply:
Fixed